src/duration-picker/DurationCard.js

```diff
@@ -101,9 +101,9 @@ export default class DurationCard extends Component {
         if (!currentUnitBlock) return null;
 
         return (
-            <div className="duration-picker">
+            <div className="param-row">
```

Comment:
OK, how about making `param-row` css a mixin , and mix into all pickers?
Is this fine?